site/src/pages/RoadmapPage/Planner.tsx

```diff
@@ -112,14 +131,42 @@ const Planner: FC = () => {
 
       // check current roadmap against last-saved roadmap in local storage
       // if they are different, mark changes as unsaved to enable alert on page leave
-      dispatch(setUnsavedChanges(localStorage.getItem('roadmap') !== roadmapStr));
+      const localRoadmap = JSON.parse(localStorage.getItem('roadmap') ?? emptyRoadmap);
+      delete localRoadmap.timestamp;
+      dispatch(setUnsavedChanges(JSON.stringify(localRoadmap) !== roadmapStr));
     }
   }, [cookies, currentPlanData, dispatch, isFirstRenderer, roadmapStr, transfers]);
 
   const { unitCount, courseCount } = calculatePlannerOverviewStats();
 
   return (
     <div className="planner">
+      <Modal
+        show={showSyncModal}
+        onHide={() => {
+          setShowSyncModal(false);
+        }}
+        className="ppc-modal"
+        centered
+      >
+        <Modal.Header closeButton>
+          <h2>Cloud roadmap out-of-date</h2>
+        </Modal.Header>
+        <Modal.Body>
+          <p>
+            Your roadmap saved locally is newer than the one saved to the cloud. Would you like to use the local one
```

Comment:
This is not very user-friendly text – I would recommend something more along the lines of:
> This device's saved roadmap is newer than the one synced to your account. Where do you want to load the roadmap from?